src/frontend/src/handler/create_function.rs

```diff
@@ -288,21 +303,30 @@ async fn download_binary_from_link(link: &str) -> Result<Bytes> {
     }
 }
 
-/// Check if the function exists in the wasm binary.
-fn check_wasm_function(runtime: &arrow_udf_wasm::Runtime, identifier: &str) -> Result<()> {
-    if !runtime.functions().contains(&identifier) {
-        return Err(ErrorCode::InvalidParameterValue(format!(
-            "function not found in wasm binary: \"{}\"\nHINT: available functions:\n  {}",
-            identifier,
-            runtime.functions().join("\n  ")
-        ))
-        .into());
-    }
-    Ok(())
+/// Get the function identifier in wasm binary.
+fn find_wasm_identifier_v2(
```

Comment:
Please add some comments on `find_wasm_identifier_v2` and `wasm_identifier_v1` to help understand why there are v1 and v2